build: trim calls into test/

The tests need quite some time to run (they are also really really
large) they should be a separate package in openSUSE.
The main package remains small and has no test/ directory, so patch
the build files not to try to recurse into there.

---
 Makefile.am  |    2 +-
 configure.ac |   16 ----------------
 2 files changed, 1 insertion(+), 17 deletions(-)

Index: libabigail-2.0/Makefile.am
===================================================================
--- libabigail-2.0.orig/Makefile.am
+++ libabigail-2.0/Makefile.am
@@ -1,5 +1,5 @@
 ## SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
-SUBDIRS = include src tools tests doc bash-completion
+SUBDIRS = include src tools doc bash-completion
 ACLOCAL_AMFLAGS = -I m4 ${ACLOCAL_FLAGS}
 headers = config.h
 
Index: libabigail-2.0/configure.ac
===================================================================
--- libabigail-2.0.orig/configure.ac
+++ libabigail-2.0/configure.ac
@@ -888,25 +888,9 @@ libabigail.pc
     doc/manuals/Makefile
   src/Makefile
   tools/Makefile
-  tests/Makefile
-    tests/data/Makefile
     bash-completion/Makefile])
 
 dnl Some test scripts are generated by autofoo.
-AC_CONFIG_FILES([tests/runtestcanonicalizetypes.sh],
-		[chmod +x tests/runtestcanonicalizetypes.sh])
-		AC_CONFIG_FILES([tests/runtestslowselfcompare.sh],
-		[chmod +x tests/runtestslowselfcompare.sh])
-AC_CONFIG_FILES([tests/mockfedabipkgdiff],
-		[chmod +x tests/mockfedabipkgdiff])
-AC_CONFIG_FILES([tests/runtestfedabipkgdiff.py],
-		[chmod +x tests/runtestfedabipkgdiff.py])
-AC_CONFIG_FILES([tests/runtestfedabipkgdiffpy3.sh],
-		[chmod +x tests/runtestfedabipkgdiffpy3.sh])
-AC_CONFIG_FILES([tests/runtestdefaultsupprs.py],
-		[chmod +x tests/runtestdefaultsupprs.py])
-AC_CONFIG_FILES([tests/runtestdefaultsupprspy3.sh],
-		[chmod +x tests/runtestdefaultsupprspy3.sh])
 
 AC_OUTPUT
 
